src/main/java/org/kohsuke/github/GHCheckRun.java

```diff
@@ -105,15 +128,22 @@ public String getHeadSha() {
     /**
      * Gets the pull requests participated in this check run.
      *
-     * @return Pull requests of this check run
+     * Note this field is only populated for events. When getting a {@link GHCheckRun} outside of an event, this is
+     * always empty.
```

Comment:
@JKalash 
Keep this in mind.  This is took me a while to figure out. I'm not sure if it is expected behavior from GitHub but it is consistent.